pclient/main.c

```diff
@@ -43,8 +55,12 @@ verify_full(void)
   unsigned long long* output = NULL;
   bool* data_items = NULL;
 
+  // Initialize nss pointer for prio
+  static nss_struct nss_pointer;
```

Comment:
Nit: no particular point in making this static if you are re-initializing it every time. I'd suggest just dropping the static, as this may be a race condition waiting to happen if the tests are parallelized.